include/dlflaux/DLFLSculpting.cc

```diff
@@ -939,7 +939,11 @@ void autoMarkEdges(DLFLObjectPtr obj) {
     // sort edge list from big to small
     DLFLEdgePtr emax, e, etmp;
     int size = edges.size();
+#ifdef _MSC_VER
+    std::vector<DLFLEdgePtr> sorted(size);
```

Comment:
visual studio compiler is pretty strict here, and does not allow array initialization with a non-const value. I converted them to std::vectors for the windows build only, but maybe it is a good idea to update this for all configurations ?

Reply:
I agree that using std::vectors would probably be best for all configs, although in the absence of proper tests I think it might be best to keep the changes to a minimum for the time being. So it sounds good to me to keep the #ifdef here and at the other comment location.